### ISSUE TITLE
Broken example CI runs

### ISSUE THREAD

**Issue by ischoegl** [Cantera/cantera#1132]
**Problem description**

An apparent upstream change causes LaTeX `\Delta` no longer to be recognized in `matplotlib` figure titles. As a result, current GH Actions for Example tests are broken.

**Steps to reproduce**

See #1127. Error message is:
```
interfaces/cython/cantera/examples/kinetics/blowers_masel.py
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/matplotlib/_mathtext.py", line 2277, in parse
    result = self._expression.parseString(s)
  File "/opt/hostedtoolcache/Python/3.9.7/x64/lib/python3.9/site-packages/pyparsing/core.py", line 1101, in parse_string
    raise exc.with_traceback(None)
pyparsing.exceptions.ParseFatalException: Unknown symbol: \Delta, found '\'  (at char 0), (line:1, col:1)
```

**System information**

GH Actions

**Additional context**

Avoiding LaTeX in examples is likely the easiest resolution (e.g. fix in #1129)

**Comment by speth** [comment]
This has already been reported (https://github.com/matplotlib/matplotlib/issues/21463) and fixed (https://github.com/matplotlib/matplotlib/pull/21454) in Matplotlib. Arguably the easiest resolution is to just wait for the next bugfix release of Matplotlib, at which point this will just go away on its own.

**Comment by ischoegl** [comment]
@12Chao just proposed an alternative fix in #1038, but letting it go away is maybe the best option as the LaTeX string is actually legitimate.

**Comment by ischoegl** [comment]
As anticipated by @speth, the issue is fixed by an upstream update of matplotlib.
